treecorr/util.py

```diff
@@ -178,6 +188,22 @@ def gen_write_fits(fits, col_names, columns, params, extname=None):
         data[name] = col
     fits.write(data, header=params, extname=extname)
 
+def gen_write_hdf(hdf, col_names, columns, params, groupname=None):
+    """Write some columns to a new FITS extension with the given column names.
+
+    :param hdf:        An open h5py.File handle. E.g. hdf = h5py.File(file_name, 'w')
+    :param col_names:   A list of columns names for the given columns.
+    :param columns:     A list of numpy arrays with the data to write.
+    :param params:      A dict of extra parameters to write in the HDF attributes.
+    :param extname:     An optional name for the extension to write. (default: None)
```

Comment:
This doc item should be groupname, not extname.

Reply:
Fixed and clarified.